common/lib/xmodule/xmodule/tests/test_stringify.py

```diff
@@ -1,22 +1,22 @@
 """
 Tests stringify functions used in xmodule html
 """
-from nose.tools import assert_equals  # pylint: disable=no-name-in-module
 from lxml import etree
+from openedx.core.lib.tests import attr
 from xmodule.stringify import stringify_children
 
 
+@attr(shard=1)
 def test_stringify():
-    shard = 1
```

Comment:
Why did you move away from the method attribute in favor of the decorator in this and `test_stringify_again` ?

Reply:
Because these are functions, not classes.  The prior code was just setting a local variable which wasn't being found by the `pytest-attrib` plugin.

Reply:
Gotcha! Thanks for clarifying.